lib/utils.js

```diff
@@ -20,6 +20,13 @@ const getQuestionAnswer = (questionKey, answerKey, ALL_QUESTIONS) => {
   return (question.answers.find(({ key }) => (key === answerKey)).value)
 }
 
+const getAnswersKeys = (questionKey, answerValues, ALL_QUESTIONS) => {
+  const question = getQuestionByKey(questionKey, ALL_QUESTIONS)
+  return answerValues.map(answerValue => {
+    return question.answers.find(({ value }) => (value === answerValue)).key
```

Comment:
Looks like a lot of parentheses at first glance, would the following also work?

return question.answers.find({ value } => value === answerValue).key

Reply:
I tried and it does not work this way.